tests/helpers/index.test.ts

```diff
@@ -26,7 +26,7 @@ describe('misc', () => {
     })
 
     it('should return propper title', () => {
-      const untitled = { ...snippet, title: undefined }
+      const untitled = { ...snippet, title: '' }
```

Comment:
[snippet type](https://github.com/xsnippet/xsnippet-web/blob/master/src/store/index.ts#L8) describes this prop as a string (required)